designs/developer/add-45/Pipelines-update/Pipelines-update.md

```diff
@@ -0,0 +1,53 @@
+---
+parent: Developer
+version: 4.5
+---
+# Add page- Updates
+
+Updates to the Add page include switching the order of the resources and pipelines section so resources come before pipelines.
+One more subtle update to the Add page is that the ‘Show pipeline visualization’ will stay open even if the user selects a new builder image or resource during their session.
+
+## Upload from Git- Updates
+
+![Add updates 1](insert image of the updated add page)
+
+A builder image is selected. When a builder image has not been selected yet, the ‘Pipelines’ section will display an info alert that says “Select a builder image and resource to see if there is a pipeline template available for this runtime.”
+
```

Comment:
For the first sentence, I would say "A builder image must be selected for a pipeline to be available. When a builder image has not..."

Reply:
I can update that wording.